d2l-icon.html

```diff
@@ -3,32 +3,43 @@
 <link rel="import" href="../d2l-colors/d2l-colors.html">
 <link rel="import" href="../d2l-polymer-behaviors/d2l-dom.html">
 <dom-module id="d2l-icon">
-	<template>
+	<template strip-whitespace>
 		<style include="d2l-colors">
 			:host {
+				-ms-flex-align: center;
+				-webkit-align-items: center;
+				align-items: center;
 				color: var(--d2l-color-tungsten);
-				display: inline-block;
-				--iron-icon-height: var(--d2l-icon-height, 18px);
-				--iron-icon-width: var(--d2l-icon-width, 18px);
```

Comment:
I'm going to make a small change to keep these variables around instead of introducing `d2l-icon-size`. That'll make this change non-breaking.